settings.py

```diff
@@ -0,0 +1,12 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "City temperature management API"
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
+    WEATHER_API_KEY: str
```

Comment:
The `WEATHER_API_KEY` is a required setting. Ensure that this key is provided in your environment or `.env` file to avoid runtime errors.